src/helpers/client/safari-client.ts

```diff
@@ -227,6 +228,15 @@ export class SafariClient implements Client {
     });
   }
 
+  /**
+   * Listens for 'isAppAlive' events.
```

Comment:
Do you mind adding a note here that the `isAppAlive` event was introduced in desktop app v1.0.9?

Reply:
Sure!

Reply:
Sure!